README.md

```diff
@@ -84,6 +84,10 @@ To use the action, add a step to your workflow that uses the following syntax.
 
   (Optional) Treat specific values as standard environment variables (unmasked).
 
+- `output-file`
+
+  (Optional) Path to file that will be populated with all `KEY=VALUE` pairs.
```

Comment:
I'm a little torn as to whether this should change the behavior and _only_ write it to disk or for it to be both.

I guess it can always be changed, for now it may be worth documenting that this is "in addition to loading them into the environment."